### ISSUE TITLE
AT-D878UV2 - Can not save and reload conf

### ISSUE THREAD

**Issue by endrec** [hmatuschek/qdmr#247]
When I read my config from my 878UV2, and save it, I get 
```
ERROR in lib/configobject.cc@304: Cannot map value 255 to enum AlertType. Ignore attribute but this points to an incompatibility in some codeplug. Consider reporting it to https://github.com/hmatuschek/qdmr/issues.
```

When I try to open the file, I get

```
ERROR in lib/configobject.cc@521: 1015:155: Cannot parse anytone of DigitalContact: Expected instance of 'AnytoneContactExtension'.
ERROR in lib/configobject.cc@1185: 1015:4: Cannot parse list.
```

Affected versions:
0.10.2 and 0.10.3 (built from b1c4d8d)

Version 0.10.1 was not affected.

**Comment by hmatuschek** [comment]
Could you attach the yaml file and maybe the binary codeplug as well? You can read the binary codeplug using the command line tool
```
dmrconfig read codeplug.dfu
```
This would help to identify the issue more easily.

**Comment by hmatuschek** [comment]
I now handle "invald" values like 0xff during the decoding of the binary codeplug. This should prevent this issue. However, I still need to figure out, where this value originates. It may point to an other issue with the codeplug. So I keep the issue open as an reminder.

**Comment by endrec** [comment]
> Could you attach the yaml file and maybe the binary codeplug as well? 

I'd rather not upload them publicly, I emailed you.

**Comment by hmatuschek** [comment]
Ok, there is an issue with the contact bank addresses. The first 1000 contacts are read correctly, everything past that is garbage.

**Comment by hmatuschek** [comment]
Just fixed the addresses of the contacts. They are organized in banks of 1000 contacts. Just tested it with all 10k contacts on a D878UV (not II) and it works now. Let me know, if it works for you.

**Comment by endrec** [comment]
Reading works for me on the D878UV II as well. :) Thanks for the quick fix!

**Comment by hmatuschek** [comment]
The fix will be part of the current release 0.10.3. Thanks for reporting it. Totally missed that.
